asmcli/lib/context.sh

```diff
@@ -0,0 +1,31 @@
+context_init() {
+  local JSON; JSON="${1}"
+  context_FILE_LOCATION="$(mktemp)"; readonly context_FILE_LOCATION
+  export context_FILE_LOCATION
+
+  echo "${JSON}" >> "${context_FILE_LOCATION}"
```

Comment:
Out of curiosity, why >> instead of >|? With `mktemp` it shouldn't matter, but personally >| feels more intuitive since we're initializing a new file and not appending to an existing one.

Reply:
I just used `>>` because the file already exists. I would change it to `>|`.